### PR TITLE
Add ability to configure where draco is installed

### PR DESCRIPTION
**Pull request by mkrus**
Support setting DRACOSDK env var to point to where Draco include and
lib folders are

### CODE REVIEW

File: kuesa.pro [hunk 1]
```diff
@@ -24,14 +24,13 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-lessThan(QT_MAJOR_VERSION, 5) | lessThan(QT_MINOR_VERSION, 11): error("This project requires Qt 5.11")
+lessThan(QT_MAJOR_VERSION, 5) | lessThan(QT_MINOR_VERSION, 12): error("This project requires Qt 5.12")
```
Comment (jjcasmar) [inline kuesa.pro]: Should this be a different patch?

Comment (krf) [reply kuesa.pro]: https://github.com/KDAB/kuesa/pull/6

Comment (mkrus) [reply kuesa.pro]: yeah, removed the change here
